pkg/gce-pd-csi-driver/controller_test.go

```diff
@@ -1413,3 +1413,76 @@ func TestPickRandAndConsecutive(t *testing.T) {
 
 	}
 }
+
+func TestVolumeOperationConcurrency(t *testing.T) {
+	snapshotToExecute := make(chan gce.SnapshotSourceAndTarget, 3)
+	readyToExecute := make(chan struct{}, 2)
+	reqFinished := make(chan error, 2)
+
+	gceDriver := initBlockingGCEDriver(t, nil, snapshotToExecute, readyToExecute)
+	cs := gceDriver.cs
+
+	vol1CreateSnapshotReqA := &csi.CreateSnapshotRequest{
+		Name:           name + "1A",
```

Comment:
What is `name` and why is it important for this test? I haven't read the other tests in this file, so you may be just going with the convention. From a github PR, finding the definition of `name` is a nuisance. I try to follow this testing philosophy when possible: https://abseil.io/tips/122

Reply:
Name is a predefined name for the snapshot. I'm just appending an identifier to the end to get unique names with the same prefix. In actuality what name really is doesn't really matter, but I'm just sticking to the convention of using name for the snapshot name as much as possible.